pybamm/util.py

```diff
@@ -57,14 +57,22 @@ def __getitem__(self, key):
         try:
             return super().__getitem__(key)
         except KeyError as error:
-            if "particle diffusivity" in key:
-                warn(
-                    f"The parameter '{key.replace('particle', 'electrode')}' "
-                    f"has been renamed to '{key}'",
-                    DeprecationWarning,
-                    stacklevel=2,
+            if "electrode diffusivity" in key or "particle diffusivity" in key:
+                old_term, new_term = (
+                    ("electrode", "particle")
+                    if "electrode diffusivity" in key
+                    else ("particle", "electrode")
                 )
-                return super().__getitem__(key.replace("particle", "electrode"))
+                alternative_key = key.replace(old_term, new_term)
+
+                if old_term == "electrode":
+                    warn(
+                        f"The parameter '{alternative_key}' has been renamed to '{key}'",
```

Comment:
```suggestion
                        f"The parameter '{alternative_key}' has been renamed to '{key}' and will be removed in a future release. Using '{key}'",
```

Would it make sense to let users know which parameter is being used regardless of the one they input?